apps/web/src/features/setup/SetupGraphic.css

```
@@ -0,0 +1,216 @@
+  /* Don't clip the scene's motion (module bob, click bursts, halos) or the
+     handoff shrink to the viewBox. */
+  overflow: visible;
+}
+
+.setup-graphic :is(.arc, .vent, .edge, .card-mid, .ring-small, .guide, .ring-large, .card-back, .card-front) {
+  stroke-linecap: round;
+}
+
+.setup-graphic
+  :is(.arc, .vent, .edge, .card-mid, .ring-small, .guide, .ring-large, .card-back, .card-front, .hairline, .panel, .knob) {
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)